int64[] missionList

int64 missionCount

int64 missionGate

int64 missionIndex

bool missionSearch

bool missionStatus

bool missionDone
